pt.html

```diff
@@ -122,25 +122,25 @@
 <panel w=400 h=650>
 <pic src="pics/intro0.png" sx=0 sy=0></pic>
 <words x=10 y=10 w=350 h=60>
-	In Greek mythology, Mnemosyne, the goddess of Memory...
+	Na mitologia grega, Mnemosine, a deusa da Memória...
 </words>
 <words x=30 y=330 w=310 h=60>
-	...was the mother of the Muses, the goddesses of inspiration.
+	...era a mãe das Musas, as deusas da inspiração.
 </words>
 <words x=70 y=445 w=90 style="color:#fff" no-bg class="comic_text">
-	music
+	música
 </words>
 <words x=119 y=494 w=90 style="color:#fff" no-bg class="comic_text">
-	theatre
+	teatro
 </words>
 <words x=214 y=464 w=90 style="color:#fff" no-bg class="comic_text">
-	weird fanfic
+	fanfic estranha
 </words>
 </panel>
 
 <panel w=600 h=60>
 <words w=600 x=-15 no-bg>
-	So, how's Memory and Inspiration doing in schools?
+	Então, como Memória e Inspiração é aplicada nas escolas?
```

Comment:
```suggestion
	Então, como Memória e Inspiração são aplicadas nas escolas?
```
Não seria plural aqui?

Reply:
Eu não tenho certeza absoluta, então vou aceitar